scripts/performance-tests.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a helper script for Knative performance test scripts.
+# See README.md for instructions on how to use it.
+
+source $(dirname ${BASH_SOURCE})/library.sh
+
+# Configurable parameters.
+# If not provided, they will fall back to the default values.
+readonly BENCHMARK_ROOT_PATH=${BENCHMARK_ROOT_PATH:-test/performance/benchmarks}
+readonly PROJECT_NAME=${PROJECT_NAME:-knative-performance}
+readonly SERVICE_ACCOUNT_NAME=${SERVICE_ACCOUNT_NAME:-mako-job}
+
+# Setup env vars.
+readonly KO_DOCKER_REPO="gcr.io/${PROJECT_NAME}"
+readonly GOOGLE_APPLICATION_CREDENTIALS="/etc/performance-test/service-account.json"
+readonly GITHUB_TOKEN="/etc/performance-test/github-token"
+readonly SLACK_READ_TOKEN="/etc/performance-test/slack-read-token"
+readonly SLACK_WRITE_TOKEN="/etc/performance-test/slack-write-token"
+
+# Set up the user for cluster operations.
+function setup_user() {
+  echo ">> Setting up user"
+  echo "Using gcloud project ${PROJECT_NAME}"
+  gcloud config set core/project ${PROJECT_NAME}
+  local user_name="${SERVICE_ACCOUNT_NAME}@${PROJECT_NAME}.iam.gserviceaccount.com"
+  echo "Using gcloud user ${user_name}"
+  gcloud config set core/account ${user_name}
+}
+
+# Update resources installed on the cluster.
+# Parameters: $1 - cluster name
+#             $2 - cluster zone/region
```

Comment:
I thought for region the gcloud arg should be `--region` instead of `--zone`

Reply:
`gcloud` is pretty smart, I tried all the four combinations `--region=us-central1`, `--zone=us-central1`, `--region=us-central1-a`, `--zone=us-central1-a`, and all of them work.

Reply:
that's a bit surprising but nice to have, I think it deserves a comment

Reply:
Done.